testing/tests/src/test/resources/golden/UnresolvedTypeTest/unresolved-types-KSP2-.golden.txt

```diff
@@ -68,13 +68,13 @@ Encountered:
              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-error: Invalid/unresolved type: kotlin.collections.List<<unresolved: ???>>
+error: Invalid/unresolved type: java.util.List<<unresolved: UnresolvedEp>>
```

Comment:
Do you have any ideas why `kotlin.collections` were replaced with `java.util`?

Reply:
`kotlin.collections` were mentioned here before the fix because when `TypeMap` encountered an error type, even if in type argument, it bailed out of mapping completely.

Now KAPT and KSP2 behave identically and correctly, while KSP1 still has some problem there.